Source/WebKit/WPEPlatform/wpe/WPEView.h

```diff
@@ -45,6 +45,7 @@ typedef struct _WPEDisplay WPEDisplay;
 typedef struct _WPEEvent WPEEvent;
 typedef struct _WPEMonitor WPEMonitor;
 typedef struct _WPERectangle WPERectangle;
+typedef struct _WPEGestureController WPEGestureController;
```

Comment:
We can include the header, since the gesture controller doesn't reference the view